package.json

```diff
@@ -18,10 +18,11 @@
     "start:automated": "webpack-dev-server --color --progress --config webpack.test.js",
     "start": "webpack-dev-server --color --progress --config webpack.dev.js",
     "example": "ws --rewrite '/* -> /dist/$1' --https",
-    "prod": "npm run build",
+    "prod": "npm run build && npm run copy",
     "snyk": "snyk test --org=secure-trading",
     "test": "jest --watchAll",
     "coverage": "jest --coverage",
+    "copy": "copy -f \"dist\\stcard.js\" \"\" && copy -f \"dist\\stcardstyle.css\" \"\"",
```

Comment:
@james-simpson-ST I'm not really good at bash/command line; sth is wrong here but I've not figure it out yet,